pkg/log/status.go

```diff
@@ -193,21 +193,39 @@ func Askf(format string, a ...interface{}) {
 	fmt.Printf("%s", bold(fmt.Sprintf(format, a...)))
 }
 
-// Status creates a spinner, sets the prefix then returns it.
+// Spinner creates a spinner, sets the prefix then returns it.
 // Remember to use .End(bool) to stop the spin / when you're done.
 // For example: defer s.End(false)
+// Spin parameter is used to start the initial spin. This helpful
+// for situations where spinning isn't viable (debug)
 func Spinner(status string) *Status {
 	s := NewStatus(os.Stdout)
 	s.Start(status, IsDebug())
 	return s
 }
 
-// Statusf creates a spinner, sets the prefix then returns it.
+// SpinnerNoSpin is the same as the "Spinner" function but forces no spinning
+func SpinnerNoSpin(status string) *Status {
+	s := NewStatus(os.Stdout)
+	s.Start(status, true)
+	return s
+}
+
+// Spinnerf creates a spinner, sets the prefix then returns it.
 // Remember to use .End(bool) to stop the spin / when you're done.
 // For example: defer s.End(false)
-func Spinnerf(format string, a ...interface{}) *Status {
+// Spin parameter is used to start the initial spin. This helpful
+// for situations where spinning isn't viable (debug)
+func Spinnerf(spin bool, format string, a ...interface{}) *Status {
```

Comment:
do we need `spin` parameter here? Shouldn't be calling `Spinnerf(false..` identical to  `SpinnerfNoSpin`?